CHANGES.md

```diff
@@ -18,6 +17,7 @@ wavesurfer.js changelog
       wavesurfer.play();
```

Comment:
Maybe in this way is clearer?
`wavesurfer.load(url | HTMLMediaElement, peaks, preload, duration);`

Reply:
True